homeassistant/components/samsungtv/__init__.py

```diff
@@ -1 +1,15 @@
 """The Samsung TV integration."""
+
+
+async def async_setup(hass, config):
+    """Set up the Samsung TV integration."""
+    return True
```

Comment:
We should probably move the config section to the integration domain key in the config yaml and import the config yaml to the config flow here.

It will be a breaking change but it will make it future proof if we add more platforms.

Reply:
@MartinHjelmare Do you mean like this?